spec/std/html_spec.cr

```diff
@@ -10,21 +10,41 @@ describe "HTML" do
     end
 
     it "escapes dangerous characters from a string" do
```

Comment:
Maybe reword to `escapes special characters from HTML string`

Reply:
Done, thanks.